### ISSUE TITLE
Libraries version mismatch on Ubuntu 16.04/18.04

### ISSUE THREAD

**Issue by johanburati** [Azure/azure-storage-fuse#247]
### Which version of the blobfuse was used?
```
$ dpkg-query -s blobfuse
Package: blobfuse
Status: install ok installed
Priority: optional
Section: devel
Installed-Size: 3987
Maintainer: Microsoft - Azure Storage
Architecture: amd64
Version: 1.0.2
Description: FUSE adapter for Azure Blob Storage - MIT License
Homepage: https://github.com/azure/azure-storage-fuse
```

### Which OS (please include version) are you using?
Tested on both **Ubuntu 16.04** and **Ubuntu 18.04**, latest marketplace images.

**Ubuntu 16.04**:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.5 LTS
Release:        16.04
Codename:       xenial
```

**Ubuntu 18.04**:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic
```

### What problem has encountered?
I have followed the install procedure at  [storage-how-to-mount-container-linux](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-how-to-mount-container-linux) but got errors related to missing libraries, I tried to fix that by  creating symlinks  and installing the missing library but looks like **blobfuse** does not like those versions.

**First attempt:**
```
$ sudo blobfuse /blobfuse --tmp-path=/mnt/blobfuse --config-file=bloblfuse.cfg -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120
blobfuse: error while loading shared libraries: libgnutls.so.26: cannot open shared object file: No such file or directory
```

**Second attempt:**
```
$ sudo ln -s /usr/lib/x86_64-linux-gnu/libgnutls.so.30 /usr/lib/x86_64-linux-gnu/libgnutls.so.26
$ sudo blobfuse /blobfuse --tmp-path=/mnt/blobfuse --config-file=bloblfuse.cfg -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120
blobfuse: error while loading shared libraries: libgcrypt.so.11: cannot open shared object file: No such file or directory
```

**Third attempt:**
```
$ sudo apt install libgcrypt11-dev
$ sudo ln -s /usr/lib/x86_64-linux-gnu/libgcrypt.so /usr/lib/x86_64-linux-gnu/libgcrypt.so.11

$ sudo blobfuse /blobfuse --tmp-path=/mnt/blobfuse --config-file=bloblfuse.cfg -o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120
blobfuse: /usr/lib/x86_64-linux-gnu/libgcrypt.so.11: version `GCRYPT_1.2' not found (required by blobfuse)
blobfuse: /usr/lib/x86_64-linux-gnu/libgnutls.so.26: version `GNUTLS_2_10' not found (required by blobfuse)
blobfuse: /usr/lib/x86_64-linux-gnu/libgnutls.so.26: version `GNUTLS_1_4' not found (required by blobfuse)

$ blobfuse -v
blobfuse: /usr/lib/x86_64-linux-gnu/libgcrypt.so.11: version `GCRYPT_1.2' not found (required by blobfuse)
blobfuse: /usr/lib/x86_64-linux-gnu/libgnutls.so.26: version `GNUTLS_2_10' not found (required by blobfuse)
blobfuse: /usr/lib/x86_64-linux-gnu/libgnutls.so.26: version `GNUTLS_1_4' not found (required by blobfuse)
```

**Comment by seguler** [comment]
I just created a brand new 18.04 image, and ran through the article and was able to install it. I suspect it is because the article is talking about 14.04 image, and asking you to modify the repository based on your Linux distro which you might have missed. I'll update the article to include each and every repository if this was the problem.

Here are all the commands I ran on the ubuntu 18.04 VM after it got initialized:

```
seguler@fuse1804:~$ history
    1  wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
    2  sudo dpkg -i packages-microsoft-prod.deb
    3  sudo apt-get update
    4  sudo apt-get install blobfuse
    5  blobfuse
    6  history
```
